clients/src/main/java/org/apache/kafka/clients/consumer/internals/ShareConsumeRequestManager.java

```diff
@@ -866,7 +876,7 @@ UnsentRequest buildRequest() {
             ShareAcknowledgeRequest.Builder requestBuilder = sessionHandler.newShareAcknowledgeBuilder(groupId, fetchConfig);
             Node nodeToSend = metadata.fetch().nodeById(nodeId);
 
-            log.trace("Building acknowledgements to send : {}", finalAcknowledgementsToSend);
+            log.info("Building acknowledgements to send : {}", finalAcknowledgementsToSend);
```

Comment:
This should not be a log.info. I see that's helpful when you're working with the code, but we should not merge this :)

Reply:
Thanks, yeah I was testing locally and pushed it by mistake. I have updated now.